src/components/common/multi-entry/multi-entry.styles.tsx

```diff
@@ -1,9 +1,23 @@
+import { Text } from "@lifesg/react-design-system/text";
 import { Accordion } from "@lifesg/react-design-system/accordion";
 import { ButtonWithIcon } from "@lifesg/react-design-system/button-with-icon";
 import { Color } from "@lifesg/react-design-system/color";
 import styled from "styled-components";
 
-export const MultiEntryAccordionItem = styled(Accordion.Item)`
+// =============================================================================
+// STYLE INTERFACE
+// =============================================================================
+interface IMultiEntryAccordionItemStyleProps {
+    $hasSubtitle: boolean;
+}
+
+export const SubtitleText = styled(Text.Body)`
```

Comment:
now that we have a divider for the interface, we can have another divider for the stylings

```suggestion
// =============================================================================
// STYLING
// =============================================================================

export const SubtitleText = styled(Text.Body)`
```